README.md

```diff
@@ -135,21 +135,24 @@ The default `Dockerfile` used by `julia_pod` is set up to
 time it takes to `using MyProject` your project.
 
 The convention for inclusion into the sysimage is that any
-project dependency with a **tight** compat bound is included,
+project dependency that is **pinned** included,
 unless it is blacklisted as a package known not to work with
-`PackageCompiler.jl`.
+`PackageCompiler.jl`. To add dependency `TheDependency` to the
+sysimage, in a julia REPL `Pkg` mode, do `]pin TheDependency`.
+Pinned packages show with a little ⚲ next to them when listed
+with `]status`.
 
 This convention came about because the package manager `Pkg.jl`
 is not aware of what dependencies are included in the sysimage
 (they are treated as un-versioned dependencies as though they
 were part of the julia std lib, and always take precedence over
```

Comment:
```suggestion
were part of the julia stdlib, and always take precedence over
```

Reply:
FYI: In the official documentation Julia refers to the language while `julia` refers to the executable. I typically use "Julia" in documentation